src/writeData/subscriptions/components/SubscriptionFormContent.tsx

```diff
@@ -85,7 +85,7 @@ const SubscriptionFormContent: FC<Props> = ({
                   )
                 }
                 status={edit ? status : ComponentStatus.Disabled}
-                maxLength={56}
+                maxLength={255}
```

Comment:
I'm not sure why this was `56` before - was that intentional?

Reply:
no idea, I think it's an age old bug